pipeline/tasks/io_tasks.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tasks related to I/O."""
+
+import base64
+import os
+import subprocess
+import shutil
+import urllib
+import zipfile
+
+from gcloud import storage
+from oauth2client.client import GoogleCredentials
+from pipeline.tasks import task_base
+
+DEFAULT_PROJECT = 'vkit-pipeline'
```

Comment:
FYI but 'vkit' is kinda our internal name, might be good to avoid for a default value in the opensource code base.  'gapic-pipeline' might be a good name, although I'm not sure we can take that project name.